.github/dependabot.yml

```diff
@@ -4,12 +4,39 @@ updates:
     directory: /
     schedule:
       interval: monthly
+    groups:
+      github-actions:
+        patterns:
+          - "*"
   - package-ecosystem: docker
     directory: /
     schedule:
       interval: monthly
+    groups:
+      docker:
+        patterns:
+          - "*"
   - package-ecosystem: gomod
     open-pull-requests-limit: 10
     directory: /
     schedule:
       interval: monthly
```

Comment:
```suggestion
      interval: weekly
```

Reply:
updated in https://github.com/aquasecurity/trivy/pull/6734/commits/4259824abc7e6a250c23d6e468025b06ae3649e1